lib/driver.js

```diff
@@ -171,9 +171,11 @@ class EspressoDriver extends BaseDriver {
       return [sessionId, caps];
     } catch (e) {
       await this.deleteSession();
-      e.message += '. If the cause might be espresso driver or server, ' +
-        'please read troubleshooting https://github.com/appium/appium-espresso-driver#contributing';
-      throw new errors.SessionNotCreatedError(e);
+      e.message += '. Check https://github.com/appium/appium-espresso-driver#troubleshooting regarding advanced session startup troubleshooting.';
+      if (e instanceof errors.SessionNotCreatedError) {
```

Comment:
instanceof might not work as expected. Prefer [isErrorType](https://github.com/appium/appium-base-driver/blob/7d0de6b6e45ea0e4c3d477f3d3675d5407757870/lib/protocol/errors.js#L749)

Reply:
ah... I saw a couple of times to replace to it in some PRs...